src/Client.php

```diff
@@ -126,28 +125,30 @@ public function getOptions(): Options
      */
     public function captureMessage(string $message, ?Severity $level = null, ?Scope $scope = null): ?EventId
     {
-        $payload = [
-            'message' => $message,
-            'level' => $level,
-        ];
+        $event = Event::createEvent();
 
```

Comment:
You can remove this blank line as the statements below are related to the line above